### ISSUE TITLE
Enable "variants-only" mode for `VariantContext2VariantConverter`

### ISSUE THREAD

**Issue by laserson** [opencb/hpg-bigdata#24]
Currently, `VariantContext2VariantConverter` will load all genotype calls from a VCF file into the resulting GA4GH `Variant` object.  Sometimes the user may only want the variant data itself, without any of the associated calls.  This should be a configurable option

**Comment by imedina** [comment]
I agree, I think this was discussed some time ago. Actually, other modes can be added, one proposal could be:
- skip-non-variants: current gVCFs contain both variants and non-variants calls, sometimes it can be useful to load only real variants and skip non-variants calls
- exclude-genotype: this is the one you meant above with 'variants-only'
